Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -194,6 +194,39 @@ extension Channel {
         }
     }
 
+    /// Configures a `ChannelPipeline` to speak HTTP/2 and sets up mapping functions so that it may be interacted with from concurrent code.
+    ///
+    /// In general this is not entirely useful by itself, as HTTP/2 is a negotiated protocol. This helper does not handle negotiation.
+    /// Instead, this simply adds the handler required to speak HTTP/2 after negotiation has completed, or when agreed by prior knowledge.
+    /// Use this function to setup a HTTP/2 server pipeline if you wish to use async sequence abstractions over inbound and outbound streams, as it allows that pipeline to evolve without breaking your code.
+    ///
+    /// - parameters:
+    ///     - mode: The mode this pipeline will operate in, server or client.
+    ///     - connectionConfiguration: The settings that will be used when establishing the connection. These will be sent to the peer as part of the
+    ///         handshake.
+    ///     - streamConfiguration: The settings that will be used when establishing new streams. These mainly pertain to flow control.
+    ///     - streamDelegate: The delegate to be notified in the event of stream creation and close.
+    ///     - position: The position in the pipeline into which to insert this handler.
+    ///     - inboundStreamInitializer: A closure that will be called whenever the remote peer initiates a new stream.
+    /// - returns: An `EventLoopFuture` containing the `AsyncThrowingStream` of inbound HTTP/2 stream channels and  the `AsyncStreamMultiplexer` inserted into this pipeline, which can be used to initiate new streams.
+    @_spi(AsyncChannel)
+    public func configureHTTP2PipelineAsync<InboundStreamChannelType>(mode: NIOHTTP2Handler.ParserMode,
+                                       connectionConfiguration: NIOHTTP2Handler.ConnectionConfiguration,
+                                       streamConfiguration: NIOHTTP2Handler.StreamConfiguration,
+                                       streamDelegate: NIOHTTP2StreamDelegate? = nil,
+                                       position: ChannelPipeline.Position = .last,
+                                       inboundStreamInitializer: @escaping @Sendable (Channel) -> EventLoopFuture<InboundStreamChannelType>) -> EventLoopFuture<(AsyncThrowingStream<InboundStreamChannelType, any Error>, NIOHTTP2Handler.AsyncStreamMultiplexer)> {
```

Comment:
This formatting is ... interesting. The parameters are aligned in two different places (`mode` vs. the-rest) and the final line is super long.

```suggestion
    public func configureHTTP2PipelineAsync<InboundStreamChannelType>(
        mode: NIOHTTP2Handler.ParserMode,
        connectionConfiguration: NIOHTTP2Handler.ConnectionConfiguration,
        streamConfiguration: NIOHTTP2Handler.StreamConfiguration,
        streamDelegate: NIOHTTP2StreamDelegate? = nil,
        position: ChannelPipeline.Position = .last,
        inboundStreamInitializer: @escaping @Sendable (Channel) -> EventLoopFuture<InboundStreamChannelType>
    ) -> EventLoopFuture<(AsyncThrowingStream<InboundStreamChannelType, any Error>, NIOHTTP2Handler.AsyncStreamMultiplexer)> {
```